cw_ethereum/lib/ethereum_client.dart

```diff
@@ -72,7 +72,7 @@ class EthereumClient {
       to: EthereumAddress.fromHex(toAddress),
       maxGas: gas,
       gasPrice: price,
-      maxPriorityFeePerGas: EtherAmount.fromInt(EtherUnit.gwei, priority.tip),
+      // maxPriorityFeePerGas: EtherAmount.fromInt(EtherUnit.gwei, priority.tip),
```

Comment:
Why do you remove the `maxPriorityFeePerGas`?

Reply:
Verifying this theory as we received multiple reports regarding this issue
https://github.com/xclud/web3dart/issues/18